Shopify/tophat
# Add support for all other Apple platforms

### What does this change accomplish?

This change adds support for watchOS, tvOS, and visionOS devices.

### How have you achieved it?

By simply adding missing switch cases. Tophat already technically supported all these platforms, but they were not enabled.

### How can the change be tested?

Grab a watchOS application bundle and try to install it.  It should install to a selected watchOS device or simulator.
